crates/noirc_evaluator/src/ssa_refactor/ssa_gen/mod.rs

```diff
@@ -16,6 +16,9 @@ use self::{
 
 use super::ir::{instruction::BinaryOp, types::Type, value::ValueId};
 
+/// Generates SSA for the given monomorphized program.
+///
+/// This function will generate the Ssa but does not perform any optimizations on it.
```

Comment:
nit: We use "SSA" and "Ssa"

Reply:
We switch in a couple of places -- though let me know if this is on purpose (we just need to have a consistent rule)

Reply:
Ok so I've edited it a bit to make it more consistent:
- `SSA` is now used in all comments with some exceptions. It is the most correct since SSA is an acronym for single-static assignment.
- `Ssa` is still used for the `struct Ssa { ... }` type since it follows the casing rules of only upper-casing the first letter in the acronym in case of acronyms. Likewise, any comments referring specifically to this type rather than the SSA concept in general can keep the Ssa spelling.
- `ssa` in lowercase is used only where all lowercase letters are expected like in function names such as `generate_ssa`. Comments in these functions should still refer to converting to "SSA" or "SSA form" in caps.